packages/editor-ui/src/mixins/workflowRun.ts

```diff
@@ -223,6 +227,14 @@ export const workflowRun = defineComponent({
 
 				const workflowData = await this.getWorkflowDataToSave();
 
+				workflowData.nodes = workflowData.nodes.map((node) => {
+					// Execute credential only nodes as HTTP Request Node
+					if (node.type.startsWith(CREDENTIAL_ONLY_NODE_PREFIX)) {
+						node.type = HTTP_REQUEST_NODE_TYPE;
```

Comment:
not enough to do this on the FE.. the BE needs to support this new type.. 
the workflow cannot be activated
<img width="835" alt="Screenshot 2023-11-08 at 13 44 59" src="https://github.com/n8n-io/n8n/assets/4711238/4d5177d5-ba43-4acd-8d0d-10873c8d41d9">

also shows as http node in execution.. (which might be forgivable but above is not)
<img width="426" alt="Screenshot 2023-11-08 at 13 46 07" src="https://github.com/n8n-io/n8n/assets/4711238/cd0d2b6d-d491-4861-8c44-55c83ac67447">

I would say it's simpler to keep the type as `n8n-nodes-base.httpRequest` and add another property 
<img width="673" alt="Screenshot 2023-11-08 at 13 55 19" src="https://github.com/n8n-io/n8n/assets/4711238/4bc0b419-77dd-4288-9484-6b96ea4630cb">

Reply:
I think the second one is because the iframe points to localhost:5678 (still running old frontend unless you rebuild)

Reply:
no, I was testing on 5678 not 8080.. as you can see in the screenshot, the type is `httpRequest` in the execution, and that's why that icon is showing.